cmd/apichecker/apicheck_test.go

```diff
@@ -0,0 +1,592 @@
+package main
+
+import (
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/thrasher-corp/gocryptotrader/common/convert"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/log"
+)
+
+var (
+	testAPIKey      = ""
+	testAPIToken    = ""
+	testChecklistID = ""
+	testCardID      = ""
+	testListID      = ""
+	testBoardID     = ""
+	testBoardName   = ""
+)
+
+func TestMain(m *testing.M) {
+	SetTestVars()
+	c := log.GenDefaultSettings()
+	c.Enabled = convert.BoolPtr(false)
```

Comment:
Set to true, otherwise no output will be output

Reply:
Please also add a comment here advising the user if they want verbose output to set it here to true